UPGRADE.md

```diff
@@ -3,6 +3,19 @@ UPGRADE
 
 # From 0.14.0 to 0.15.x
 
+## Media Bundle
+Created a bundle named `SyliusMediaBundle` which is helping to integrate CMF Media image documents into ORM entities.
+
+https://github.com/Sylius/Sylius/pull/2975
+
+#### Codebase upgrade:
+ * Now we're using CMF Media's `Image` *document*.
+ * SyliusMediaBundle offers an `Image` *entity* which is the one developers should use, since they cannot directly create a relationship from an entity to a document.
+ * ProductVariantImage entity is removed.
+ * ProductVariant, Taxon and Taxonomy images are now having a relationship with `Image` *entity*.
+ * Form type for `Image` fields is `sylius_image` not `cmf_media_image`. Actually we're doing some extra work with `cmf_media_image`, if you do that yourself it can be OK to use it.
+ * Since we're using CMF Media Bundle upload helper, every rule from that bundle is applied here as well. (e.g. File name cannot contain a space character)
+
```

Comment:
All this should to changelog. UPGRADE file is about what you need to do in order to upgrade. :)